src/Controller/EventsController.php

```diff
@@ -103,11 +103,73 @@ public function show($id, $slug, Request $request)
         } catch (NotFoundException $e) {
             throw new NotFoundHttpException('Event not found', $e);
         }
-
+        
         $data = [
-            'event'       => $event
+            'event'       => $event,
+            'schema'      => $this->createEventSchemaJSON($event)
         ];
 
         return $this->render('events/show.html.twig', $data);
     }
+
+    private function createEventSchemaJSON($event) {
+        $content = array($event->getContent());
+       
+        $online = array(
+            "@type" => "VirtualLocation",
+            "url" => ($content[0]['cta_destination']->getValue())
+        );
+            
+        $in_person = array(
+            "@type" => "Place",
+            "name" =>  $content[0]['place_name']->getValue(),
+            "address" => array(
+                "@type" => "PostalAddress",
+                "streetAddress" => $content[0]['street_address']->getValue(),
+                "addressLocality" => $content[0]['address_locality']->getValue(),
+                "postalCode" => $content[0]['postal_code']->getValue(),
+                "addressRegion" => $content[0]['address_region']->getValue(),
+                "addressCountry" => $content[0]['address_country']->getValue()
+            )
+        );
+
+        $location = [];
+        $hybrid = array($online, $in_person);
```

Comment:
This $hybrid only get used once so we can just do the assignment on line151 inside the case statement